### ISSUE TITLE
2.5.2 Pointer Cancellation and Success Technique Matching the accessible name to the visible label

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/wcag#888]
Could someone explain how the Success Technique '**[Matching the accessible name to the visible label](https://www.w3.org/WAI/WCAG21/Techniques/general/G211.html)** fits as a Success Technique for WCAG 2.1 SC **[2.5.2 Pointer Cancellation](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.1&showtechniques=252#pointer-cancellation)** Level A

 I believe this is incorrectly placed and should be removed.

**Comment by awkawk** [comment]
This was a mistake and is fixed in the editor's draft, to be pushed live tomorrow.
